packages/ts-client/src/webrtc/NegotiationManager.ts

```diff
@@ -0,0 +1,7 @@
+export class NegotiationManager {
+  /**
+   * Indicates if an ongoing renegotiation is active.
+   * During renegotiation, both parties are expected to actively exchange events: renegotiateTracks, offerData, sdpOffer, sdpAnswer.
+   */
+  public ongoingRenegotiation: boolean = false;
```

Comment:
WDYT about having two functions `renegotationStarted()` and `renegotationCompleted()` with private variable?

Reply:
I agree. That's why I created this class. I will do it in future PRs